src/GUI/UI/Houses/Shops.cpp

```diff
@@ -861,6 +861,18 @@ void GUIWindow_Shop::playHouseGoodbyeSpeech() {
     }
 }
 
+void GUIWindow_AlchemyShop::playHouseGoodbyeSpeech() {
+    if (pParty->PartyTimes.shopBanTimes[houseId()] <= pParty->GetPlayingTime()) {
```

Comment:
Let's add a comment here --- why are we adding special processing for alchemy shop in the first place.